api/src/endpoints/dataOverlay.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express';
+import { readFileSync } from 'fs';
+
+const routes = express.Router();
+
+routes.get('/:model', async (req, res) => {
+  const { model } = req.params;
+  try {
+    const indexJson = readFileSync(
+      `/project/dataOverlay/${model}/index.json`,
```

Comment:
Would it be possible to avoid hard-coding the `/project` folder name, e.g. by using a relative path?

Reply:
The relative path might be a bit difficult since I think they would be different for `local` and `prod`. It should be possible but I'm not sure if the implementation would be less rigid than the current one. I would also consider having an environment variable to replace `/project/`. If we do decide to change anything, [this file](https://github.com/MetabolicAtlas/MetabolicAtlas/blob/develop/api/src/neo4j/queries/map.js#L9) should also be updated.

Reply:
The name `/project` is decided in a single place:
https://github.com/MetabolicAtlas/MetabolicAtlas/blob/ecba3f35ddc1acbfeaa8679a600a875f642db6b2/api/Dockerfile#L3
and so both the `local` and `prod` docker-compose files use the same folder name.

Reply:
If we want to use a relative path, I think `local` is in `src/endpoints` and `prod` is in `dist`.

Reply:
> If we want to use a relative path, I think `local` is in `src/endpoints` and `prod` is in `dist`.

😕  sorry I don't get it

Reply:
Maybe I got confused before. Did you mention the `WORKDIR` to use in the absolute file path? If so then it makes sense but if it's still about the relative path I'm not sure how it helps.

Reply:
The suggestion below seems to work:

```suggestion
      `./dataOverlay/${model}/index.json`,
```

Reply:
Ahh right, in this case it does work since the file reading happens from the place of execution `api`. I thought it would be like when we do a `require` or `import` like `../../dataOverlay` for `local` and `../dataOverlay` for `prod`.